src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -24,10 +24,24 @@ public static String buildSearchConceptsQuery(
       TableNameGenerator tableNameGenerator,
       CloudPlatformWrapper platform) {
     var conceptTablePointer = TablePointer.fromTableName("concept", tableNameGenerator);
+    var conditionOccurrencePointer =
+        TablePointer.fromTableName("condition_occurrence", tableNameGenerator);
```

Comment:
Can this use the domain to determine which occurrence table to use? Different concept domains will have different occurrence tables.

Reply:
Let me look into it :)

Reply:
would this require the usage of `DOMAIN_TO_OCCURRENCE_TABLE` from `CriteriaQueryBuilder.java`?